app/views/users/new.html.erb

```diff
@@ -1,32 +1,10 @@
-<% provide :title, t("sign_up") %>
-<h1><%= t("sign_up") %></h1>
+<% provide :title, t(:sign_up) %>
+<% provide(:button_text, t(:create_my_account)) %>
+<h1><%= t(:sign_up) %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for @user, url: signup_path do |f| %>
-
-      <% if @user.errors.any? %>
-        <div id="error_explanation">
-          <div class="alert alert-danger">
-            <%= t "the_form_contains" %>
-            <%= pluralize @user.errors.count, "error" %>.
-          </div>
-          <ul>
-          <% @user.errors.full_messages.each do |msg| %>
-              <li><%= msg %></li>
-            <% end %>
-          </ul>
-        </div>
-      <% end %>
-
-      <%= f.label :name %>
-      <%= f.text_field :name %>
-      <%= f.label :email %>
-      <%= f.email_field :email %>
-      <%= f.label :password %>
-      <%= f.password_field :password %>
-      <%= f.label :password_confirmation, t("confirmation") %>
-      <%= f.password_field :password_confirmation %>
-      <%= f.submit t("create_my_account"), class: "btn btn-primary" %>
+    <%= render "form" %>
     <% end %>
```

Comment:
remove this line

Reply:
hi i already editted this file, but after i push, it didnt show in github
![image](https://user-images.githubusercontent.com/87749692/131635704-f537549f-dd17-4806-b232-c83a12d64552.png)

Reply:
The line with red color meaning you deleted it

Reply:
then should i not deleted it ? and rewrite again ??